packages/google_sign_in/google_sign_in_web/lib/google_sign_in_web.dart

```diff
@@ -53,14 +58,16 @@ class GoogleSignInPlugin extends GoogleSignInPlatform {
     }
   }
 
-  /// A future that resolves when both GAPI and Auth2 have been correctly initialized.
+  /// A future that resolves when the SDK has been correctly loaded.
   @visibleForTesting
   Future<void> get initialized {
     _assertIsInitCalled();
-    return Future.wait(<Future<void>>[_isGapiInitialized, _isAuthInitialized]);
+    return _isJsSdkLoaded;
   }
 
-  String? _autoDetectedClientId;
+  /// Stores the client ID if it was set in a meta-tag of the page.
+  @visibleForTesting
+  late String? autoDetectedClientId;
```

Comment:
This was me attempting to get the following behavior:

final, nullable and defaults to null if not set, but this actually throws an error if accessed before it's set (rather than defaulting to null), so I'm going to make this a private with only a public getter, and remove `late`.

Reply:
(So it can still be `null` after the initialization completes)